README.md

```diff
@@ -105,6 +105,18 @@ Please check [Tips for using Azure CLI effectively](https://docs.microsoft.com/e
 
 For more usage examples, take a look at our [GitHub samples repo](http://github.com/Azure/azure-cli-samples) or [https://docs.microsoft.com/cli/azure/overview](https://docs.microsoft.com/cli/azure/overview).
 
+### Write and run commands in Visual Studio Code
+
+With the [Azure CLI Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azurecli) Visual Studio Code extension, you can create `.azcli` files and use these features:
```

Comment:
Not only `.azcli`, `.ps1` is also supported! https://github.com/Azure/azure-cli/issues/17216